.github/workflows/publish_from_package.yml

```diff
@@ -0,0 +1,71 @@
+# This workflow can be manually triggered to publish all of the plugins
+# that have a version in their package.json that is newer than what
+# is available in the npm registry. This is useful if lerna publishing
+# failed after a package had its version updated on github but before
+# that version was published to npm.
+
+name: publish unpublished plugins
```

Comment:
```suggestion
name: Publish unpublished plugins
```

For consistency.

Reply:
This was done from merging in master, I don't want to muddy up actual changes into what should be a merge commit. Anyway, the publish action was already lower case so it's not really any more inconsistent than publish already is